src/test/test_interval_bitwise.cpp

```
@@ -20,3 +22,16 @@ TEST_CASE("bitwise_and with non-singleton containing all-ones includes zero", "[
     const Interval anomalous_right{-5, 5};
     REQUIRE(left.bitwise_and(anomalous_right) == Interval{0, 100});
 }
+
+TEST_CASE("finite domain left shift by zero preserves 64-bit interval", "[finite_domain][bitwise]") {
+    FiniteDomain::clear_thread_local_state();
+    const auto r1 = reg_pack(1);
```

Cppcheck (2.20.0)

[style] The function 'make_jump' is never used.

(unusedFunction)